### PR TITLE
Get user details from OpenShift

### PR DESCRIPTION
**Pull request by marcosmamorim**

### CODE REVIEW

File: operator/user_namespace.py [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+    def namespace_annotations(self):
+        return self.namespace_metadata.get('annotations', {})
+
+    @property
+    def namespace_labels(self):
+        return self.namespace_metadata.get('labels', {})
+
+    @property
+    def requester(self):
+        return self.namespace_annotations.get(f'{openshift_domain}/requester')
```
Comment (jkupferer) [inline operator/user_namespace.py:40]: The openshift domain is fixed as `openshift.io` and very unlikely to every change. Don't think we really need a variable for it?

File: operator/user_namespace.py [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+    @property
+    def user_annotations(self):
+        return self.user_metadata.get('annotations', {})
+
+    @property
+    def user_labels(self):
+        return self.user_metadata.get('labels', {})
+
+    @property
+    def user_work_id(self):
```
Comment (jkupferer) [inline operator/user_namespace.py:68]: This looks unused... If we were doing this it really should keep the naming to correspond with `rhatWorkerId` or `rhat_worker_id` so it is clear what it is.

File: operator/user_namespace.py [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+    @staticmethod
+    def get_user_info(username):
+        custom_objects_api = kubernetes.client.CustomObjectsApi()
+        try:
+            return custom_objects_api.get_cluster_custom_object(
+                openshift_user_domain, openshift_user_version, 'users', username
+            )
+        except kubernetes.client.rest.ApiException as e:
+            if e.status == 404:
+                dict()
```
Comment (jkupferer) [inline operator/user_namespace.py:51]: Did you mean something like `return {}` ?

**Review by jkupferer — CHANGES_REQUESTED** [review]

File: operator/user_namespace.py (hunk 3 above)
Comment (marcosmamorim) [reply operator/user_namespace.py]: Yes, good catch!!!

**Pull request merged**